docs/development/developing/spark/metrics.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Spark Integration Metrics
+sidebar_position: 2
+---
+
+# Spark Integration Metrics
+
+The OpenLineage integration with Spark not only utilizes the Java client's metrics but also introduces its own set of metrics specific to Spark operations. Below is a list of these metrics.
```

Comment:
Why is this in development section? This may be interesting for all the Spark users I believe, not only developers willing to extend spark integration.

Reply:
Moved to user section.